dsgrid/config/common.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from dsgrid.dimension.base_models import DimensionType
+
+
+def make_base_dimension_template(
+    exclude_dimension_types: set[DimensionType] | None = None,
```

Comment:
Should this default to `{DimensionType.TIME}`?

Reply:
You should generally never set a default function argument to a mutable instance because that argument is only evaluated once.